### PR TITLE
fix(angular/checkbox-panel): correct spacing in checkbox-panel

### LINKED ISSUE #1456: Bug: rezising layout issue with <sbb-radio-button-panel> combined with <sbb-radio-button-panel-note> containing an icon
**Describe the bug**
When reducing the window at some point the radio button and its label defined by <sbb-radio-button-panel> are no longer aligned with the icon within <sbb-radio-button-panel-note> on the Y axis (vertically). The radio button appears to have top alignement while the icon is still in the middle of the panel. A margin at the bottom of the "sbb-selection-inner-content" is being added.

**Expected behavior**
The alignement of the radio button and its label should remain in the middle of the panel even if the window size is reduced.

**Screenshots**
ok: 
image
with reduced window size: nok:
image

**Desktop (please complete the following information):**
 - OS: Window
 - Browser: chrome
 - Version: 101.0.4951.54

**Additional context**

### PR DESCRIPTION
**Pull request by mhaertwig**
- Fix vertical spacing in checkbox-panel and radio-button-panel on small devices.

- Adjust spacing of `checkbox-panel-warning` according to [design specification](https://previewserver.sbb.ch/projects/SBBCH/2022-03-16_15-48-37_8e2fe31b7b069c0185eed602135b40f72f44bb13/prod/webshop/modules/wm22_-_webshop_checkbox_radio_panel/wm22_-_webshop_checkbox_radio_panel.html): On the [example page](http://localhost:4200/angular/components/checkbox-panel/examples) the "Reservation not possible" text can now wrap (see spec). This is to avoid breaking the style on small devices that are a bit bigger than a tablet.
image

Fixes #1456

### CODE REVIEW

**Review by kyubisation — APPROVED** [review]
LGTM 👍

**Comment by mhaertwig** [comment]
The problem in your example is that the `@media` Queries don't apply because of the sidebar taking its space. I did not manage to solve this yesterday. However today...
image

File: src/angular/styles/_typography.scss [hunk 1]
```diff
@@ -1878,10 +1878,9 @@ textarea {
   color: var(--sbb-color-grey);
 
   @include sbb.mq($from: tablet) {
-    padding-left: calc(#{sbb.pxToRem(24)} * var(--sbb-scaling-factor));
-    padding-right: calc(#{sbb.pxToRem(8)} * var(--sbb-scaling-factor));
-    flex-basis: auto;
+    flex-basis: 0%;
```
Comment (jeripeierSBB) [inline src/angular/styles/_typography.scss:1881]:
```suggestion
    flex-basis: 0;
```

**Review by jeripeierSBB — APPROVED** [review]
LGTM

**Pull request merged**